### PR TITLE
dhcpcd-9.4.0: Fix infinite DHCP INFORM messages

### PR DESCRIPTION
**Pull request by klanmiko**
When an address is configured on the interface, do not send a DHCP
INFORM message if the interface is already bound (ack already received)

The second issue is that dhcp_inform resets the lease information, and
may drop the existing server address.

#47

### CODE REVIEW

File: src/dhcp.c [hunk 1]
```diff
@@ -2619,7 +2619,6 @@ dhcp_inform(struct interface *ifp)
 	    &ia->addr, &ia->mask);
 	if (state->offer_len) {
 		dhcp_new_xid(ifp);
-		get_lease(ifp, &state->lease, state->offer, state->offer_len);
```
Comment (rsmarples) [inline src/dhcp.c:2622]: Why was this removed? Doesn't look right.

Comment (klanmiko) [reply src/dhcp.c:2622]: This line resets the server address, forces a broadcast out. Should we broadcast or use the open UDP socket? I need to do additional testing of what side effects deleting this line might have.

**Comment by geki-yaba** [comment]
Thanks for this patch. This helps with a mesh setup of APs. I often have "reconnects", which puts me offline for some time - quite annoying. Dunno what it internally does?! Maybe AP transition.. And with this patch, I no longer have these down times.

Just a heads up for usefulness of this patch. :)
